src/idpyoidc/server/oauth2/token_helper.py

```diff
@@ -618,6 +619,25 @@ def _validate_configuration(self, config):
                 f"Unsupported default requested_token_type {_default_requested_token_type}"
             )
 
+    def get_handler_key(self, request, endpoint_context):
+        if "token_exchange" in endpoint_context.cdb[request["client_id"]]:
+            try:
+                default_requested_token_type = endpoint_context.cdb[request["client_id"]][
+                    "token_exchange"]["default_requested_token_type"]
+            except KeyError:
+                try:
+                    default_requested_token_type = self.config["default_requested_token_type"]
+                except:
```

Comment:
I think this is how it was written before. But this can change to the following and no try/except would be needed

```py
default_requested_token_type = self.config.get("default_requested_token_type", DEFAULT_REQUESTED_TOKEN_TYPE)
```

Reply:
@c00kiemon5ter see my next comment